### ISSUE TITLE
Build inconsistent with different .NET SDK versions installed (example: .NET 8 RC2)

### ISSUE THREAD

**Issue by rjmurillo** [microsoft/VirtualClient#212]
While running a build with .NET 8 RC2 SDK installed, the build.cmd can fail. Specifying the SDK tool chain in `global.json` to use .NET 7 SDK resolves this issue. See #211 for fix.

```
C:\Program Files\dotnet\sdk\8.0.100-rc.2.23502.2\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(106,33): error MSB4030: "true -p:TrimUnusedDependenc
ies=true" is an invalid value for the "PublishTrimmed" parameter of the "ProcessFrameworkReferences" task. The "PublishTrimmed" parameter is of type "System.Boolean". [E:\src\GitHub\rjmur
illo\VirtualClient\src\VirtualClient\VirtualClient.Dependencies\VirtualClient.Dependencies.csproj]
C:\Program Files\dotnet\sdk\8.0.100-rc.2.23502.2\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(106,33): error MSB4030: "true -p:TrimUnusedDependenc
ies=true" is an invalid value for the "PublishTrimmed" parameter of the "ProcessFrameworkReferences" task. The "PublishTrimmed" parameter is of type "System.Boolean". [E:\src\GitHub\rjmur
illo\VirtualClient\src\VirtualClient\VirtualClient.Contracts\VirtualClient.Contracts.csproj]
C:\Program Files\dotnet\sdk\8.0.100-rc.2.23502.2\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(106,33): error MSB4030: "true -p:TrimUnusedDependenc
ies=true" is an invalid value for the "PublishTrimmed" parameter of the "ProcessFrameworkReferences" task. The "PublishTrimmed" parameter is of type "System.Boolean". [E:\src\GitHub\rjmur
illo\VirtualClient\src\VirtualClient\VirtualClient.Monitors\VirtualClient.Monitors.csproj]
C:\Program Files\dotnet\sdk\8.0.100-rc.2.23502.2\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(106,33): error MSB4030: "true -p:TrimUnusedDependenc
ies=true" is an invalid value for the "PublishTrimmed" parameter of the "ProcessFrameworkReferences" task. The "PublishTrimmed" parameter is of type "System.Boolean". [E:\src\GitHub\rjmur
illo\VirtualClient\src\VirtualClient\VirtualClient.Api\VirtualClient.Api.csproj]
C:\Program Files\dotnet\sdk\8.0.100-rc.2.23502.2\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(106,33): error MSB4030: "true -p:TrimUnusedDependenc
ies=true" is an invalid value for the "PublishTrimmed" parameter of the "ProcessFrameworkReferences" task. The "PublishTrimmed" parameter is of type "System.Boolean". [E:\src\GitHub\rjmur
illo\VirtualClient\src\VirtualClient\VirtualClient.Common\VirtualClient.Common.csproj]
C:\Program Files\dotnet\sdk\8.0.100-rc.2.23502.2\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(106,33): error MSB4030: "true -p:TrimUnusedDependenc
ies=true" is an invalid value for the "PublishTrimmed" parameter of the "ProcessFrameworkReferences" task. The "PublishTrimmed" parameter is of type "System.Boolean". [E:\src\GitHub\rjmur
illo\VirtualClient\src\VirtualClient\VirtualClient.Actions\VirtualClient.Actions.csproj]
C:\Program Files\dotnet\sdk\8.0.100-rc.2.23502.2\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(106,33): error MSB4030: "true -p:TrimUnusedDependenc
ies=true" is an invalid value for the "PublishTrimmed" parameter of the "ProcessFrameworkReferences" task. The "PublishTrimmed" parameter is of type "System.Boolean". [E:\src\GitHub\rjmur
illo\VirtualClient\src\VirtualClient\VirtualClient.Core\VirtualClient.Core.csproj]
C:\Program Files\dotnet\sdk\8.0.100-rc.2.23502.2\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(106,33): error MSB4030: "true -p:TrimUnusedDependenc
ies=true" is an invalid value for the "PublishTrimmed" parameter of the "ProcessFrameworkReferences" task. The "PublishTrimmed" parameter is of type "System.Boolean". [E:\src\GitHub\rjmur
illo\VirtualClient\src\VirtualClient\VirtualClient.Main\VirtualClient.Main.csproj]
```

**Comment by yangpanMS** [comment]
@rjmurillo we plan to skip .NET 7 and go directly to .NET 8 after the release in .NET conf

**Comment by rjmurillo** [comment]
@yangpanMS #211 updated to specify .NET 6 tool chain instead of 7. This is required if you're running .NET 8 SDK as the build breaks without it.

**Comment by yangpanMS** [comment]
Thanks Richard. I approved. Most of our team is still on .NET 8. When moving to .net 8 I will address the build breaks.

**Comment by yangpanMS** [comment]
Closing as it is addressed in #214
